common4j/src/main/com/microsoft/identity/common/java/authorities/Authority.java

```diff
@@ -134,7 +135,12 @@ public static Authority getAuthorityFromAuthorityUrl(String authorityUrl) {
 
         final List<String> pathSegments = authorityCommonUriBuilder.getPathSegments();
 
-        if (pathSegments.size() == 0) {
+        // Adding check in case we have a trailing "/" at the end of the authority
+        if (pathSegments.size() == 0 || (pathSegments.size() == 1 && pathSegments.get(0).equals(""))) {
```

Comment:
I'm not sure if this is really needed or preferred, even. Basically checking if they passed `https://tenant.ciamlogin.com/` instead of just `https://tenant.ciamlogin.com`. Maybe it's ok to simply expect devs to pass `https://tenant.ciamlogin.com`, and whatever behavior occurs with `https://tenant.ciamlogin.com/` is out of scope. Thoughts?

Reply:
let's be flexible on the trailing slash. It does not cost much to be.

Reply:
Done, this format is supported